backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -109,9 +115,12 @@ public Token getNewToken(UUID shopId) {
 	// Method returns all shops keeping in mind the feature of restoring shops later
 	// on.
 	public GetShopsByShopOwnerResponse getShopsByShopOwner(String shopOwnerId) {
-		List<Shop> shops = shopMap.entrySet().stream()
-				.filter(map -> map.getValue().getShopOwnerId().equals(shopOwnerId)).map(map -> map.getValue())
-				.collect(Collectors.toList());
+		List<ShopInfo> shops = new ArrayList<>();
+		shopMap.entrySet().stream().forEach(shop -> {
+			if (shop.getValue().getShopOwnerId().equals(shopOwnerId)) {
+				shops.add(new ShopInfo(shop.getValue(), getCustomersInQueue(shop.getValue().getShopId())));
```

Comment:
Nit. try to use 'collect' of java streams here.

Reply:
done!